src/android/SplashScreen.java

```diff
@@ -194,9 +263,25 @@ public void run() {
                 // TODO: Use the background color of the webView's parent instead of using the
                 // preference.
                 root.setBackgroundColor(preferences.getInteger("backgroundColor", Color.BLACK));
+                
                 root.setLayoutParams(new LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT,
                         ViewGroup.LayoutParams.MATCH_PARENT, 0.0F));
-                root.setBackgroundResource(drawableId);
+                
+                // Use an ImageView to render the image because of its flexible scaling options.
+                splashImageView = new ImageView(context);
+                splashImageView.setImageResource(drawableId);
```

Comment:
Do you know of any other reason for setting this background color? If not, then I think we can take out Linear Layout.